src/bin-support.js

```diff
@@ -56,10 +57,14 @@ async function runTemplateTransform(root, transformName, args) {
   try {
     let transformPath = getTransformPath(root, transformName);
     let binOptions = ['-t', transformPath, ...paths];
+    let templateRecastBinPath = path.join(
+      path.dirname(require.resolve('ember-template-recast/package.json')),
+      'bin',
+      'ember-template-recast.js'
+    );
```

Comment:
I think we need to get the bin path from the package.json

ember-template-recast changed where its `bin` script lives in the 4.x series:

https://github.com/ember-template-lint/ember-template-recast/blob/v4.3.0/package.json#L23

https://github.com/ember-template-lint/ember-template-recast/blob/v4.2.1/package.json#L23

---

I think we can accommodate by:

```suggestion
    let templateRecastDir = path.dirname(require.resolve('ember-template-recast/package.json'));
    let templateRecastPkg = require('ember-template-recast/package');
    let templateRecastBinPath = path.join(templateRecastDir, templateRecastPkg.bin);
```